src/cordova/build.action.mjs

```diff
@@ -81,6 +86,23 @@ export async function main(...parameters) {
   }
 }
 
+async function checkSdkFolderIsWritable() {
+  const sdkFolder = process.env.ANDROID_SDK_ROOT;
+  if (!isFolderWritable(sdkFolder)) {
+    console.error(`The sdk folder '${sdkFolder}' is not writable! Android builds will fail.`);
+    const uid = os.userInfo().uid;
+    if (uid !== 0) {
+      console.error(`It might be non-writable because you are running as ${uid} instead of running as root.`);
+      console.error(`A workaround is to run 'node ./src/cordova/build.action.mjs' instead of 'npm run action cordova/build'`);
```

Comment:
For some reason `./tools/build.sh npm run action cordova/build` drop to a user account inside the container,

while `./tools/build.sh node ./src/cordova/build.action.mjs` runs as root.

If we can make the first one run as root, then this commit could probably be removed/reverted.